src/components/EditHospitalForm/index.js

```diff
@@ -5,17 +5,15 @@ import Heading from "../Heading";
 import Input from "../Input";
 import Label from "../Label";
 import Form from "../Form";
-import validateUrl from "../../helpers/validateUrl";
 import isPresent from "../../helpers/isPresent";
+import SelectStatus from "../../components/SelectStatus";
 
 const EditHospitalForm = ({ errors, setErrors, hospital = {}, submit }) => {
   const [hospitalName, setHospitalName] = useState(hospital.name);
-  const [hospitalSurveyUrl, setHospitalSurveyUrl] = useState(
-    hospital.surveyUrl
-  );
-  const [hospitalSupportUrl, setHospitalSupportUrl] = useState(
-    hospital.supportUrl
-  );
+  const [hospitalStatus, setHospitalStatus] = useState(hospital.status);
+  const [hospitalCode, setHospitalCode] = useState(hospital.code);
+
+  const action = hospital.id ? "Edit" : "Add";
```

Comment:
It's best to avoid the ternary operator for things like this as it makes things unclear

Reply:
This was actually the previous code, which I re-added to the top of the code block. I think they wanted to combined two actions (Edit and Add) in one. How should I change the code? Should I add an if statement? Please suggest. Thanks.

Reply:
Yeah an `if` would be better

Reply:
If statement added. Please review, thank you.